src/test/java/com/endava/service/IncomeExpenseServiceImplTest.java

```diff
@@ -210,22 +206,21 @@ public void testViewIncomesOnIncorrectDate() {
 
         Date date = new Date(System.currentTimeMillis() + 1000000000L);
 
-        incomeExpenseService.viewIncomesOn(date);
+        incomeExpenseService.getMoneyTransferOn(date);
 
-        verify(incomeDao, never()).readAll();
+        verify(moneyTransferDao, never()).readAll();
     }
 
     @Test
     public void testViewIncomesLastWeek() {
-        List<Income> incomes = new ArrayList<>( Arrays.asList(income1, income2, income3, income4) );
-        Date date = new Date(System.currentTimeMillis() - (7 * DAY_IN_MS));
+        List<MoneyTransfer> moneyTransfers = new ArrayList<>( Arrays.asList(moneyTransfer, moneyTransfer1, moneyTransfer2, moneyTransfer3) );
 
-        when(incomeDao.readAll()).thenReturn(incomes);
+        when(moneyTransferDao.readAll()).thenReturn(moneyTransfers);
 
-        List<Income> actualIncomes = incomeExpenseService.viewIncomesLastWeek();
-        List<Income> expectedIncomes = new ArrayList<>( Arrays.asList(income1, income2) );
+        List<MoneyTransfer> actualIncomes = incomeExpenseService.getMoneyTransferLastWeek();
+        List<MoneyTransfer> expectedIncomes = new ArrayList<>( Arrays.asList(moneyTransfer, moneyTransfer1) );
```

Comment:
You don't need new ArrayList here. Arrays.asList returns unmodifiable List<T>, right what you need :)